pkgs/by-name/ai/aider/package.nix

```diff
@@ -0,0 +1,97 @@
+{
+  lib,
+  python3,
```

Comment:
Also, this currently won't build with python 3.12, at least with the llama deps. So we'd need `python311` here.

Reply:
Currently it won't build for 3.12 because of tensorflow, specifically.